src/Traits/HasPermissions.php

```diff
@@ -7,20 +7,44 @@
 trait HasPermissions
 {
     /**
-     * Grant the given permission to a role.
+     * Grant the given permission(s) to a role.
      *
-     * @param  $permission
+     * @param string|array|Permission|\Illuminate\Support\Collection $permissions
+     * @param array $params
      *
      * @return HasPermissions
      */
-    public function givePermissionTo($permission)
+    public function givePermissionTo($permissions, ...$params)
     {
```

Comment:
I could ofcourse, here is the ripple effect though.
The method no longer requires you to pass a parameter to givePermissionTo()  even though we do need atleast one. Therefore I would need to write at least one more test to ensure we have one argument if not Throw a proper exception.

If you like it that way I will implement it, any suggestions on the Exception and Exception message?